tooling/v8-snapshot/src/setup/force-no-rewrite.ts

```diff
@@ -58,4 +58,11 @@ export default [
   'packages/server/lib/open_project.ts',
   'packages/server/lib/project-base.ts',
   'packages/server/lib/socket-ct.ts',
+  'node_modules/prettier/index.js',
+  'node_modules/prettier/parser-babel.js',
+  'node_modules/prettier/parser-espree.js',
+  'node_modules/prettier/parser-flow.js',
+  'node_modules/prettier/parser-meriyah.js',
+  'node_modules/prettier/parser-typescript.js',
+  'node_modules/prettier/third-party.js',
```

Comment:
how do we know this won't change under the hood with new versions of `prettier` or if we use new imports from `prettier`?

Reply:
Unfortunately, we don't have a mechanism to verify this today. cypress in cypress testing does not use snapshots due to how the child cypress server is being launched. This is something I want to investigate a solution to ASAP, but for now I think it's worth getting this fix in.

Reply:
Does this need to be manually verified at release then for each release?

Reply:
I think we would only need to verify it if we update the version of prettier that is being used: https://github.com/cypress-io/cypress/blob/develop/packages/data-context/package.json#L45